<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>MmsEditAccountBase</class>
 <widget class="QDialog" name="MmsEditAccountBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>219</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MMS Account</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Network Profile:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="networkCombo" />
   </item>
   <item>
    <widget class="QPushButton" name="networkBtn" >
     <property name="text" >
      <string>Configure Network...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="autoRetrieve" >
     <property name="text" >
      <string>Retrieve Automatically</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>201</width>
       <height>141</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
